tinyagentos/peer.py

```
@@ -0,0 +1,215 @@
+
+    try:
+        conn = sqlite3.connect(str(hub_db))
+        row = conn.execute(
+            "SELECT username FROM hub_authors WHERE fingerprint = ?",
+            (fp,),
+        ).fetchone()
+        conn.close()
+        if row and row[0]:
+            return f"hub:{row[0]}"
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)